execute/executetest/result.go

```diff
@@ -45,3 +48,47 @@ func (ti *TableIterator) Do(f func(flux.Table) error) error {
 	}
 	return nil
 }
+
+// EqualResults compares two lists of Flux Results for equlity
+func EqualResults(want, got []flux.Result) (bool, error) {
```

Comment:
Where is this function used?

Reply:
It will be used in platform for the generated ht tests.